src/services/aws/registration/registration.ts

```diff
@@ -23,6 +24,7 @@ const registration_api = aws.injectEndpoints({
         url: `${v(4)}/registration`,
         method: "POST",
         body: { Email: email },
+        headers: { authorization: TEMP_JWT },
```

Comment:
getting jwt token from current session is async but headers per endpoints can only be defined sync so set `TEMP_JWT` which will be replaced when hits `service.prepareHeaders`